src/vnsw/agent/contrail/windows/msi/agent_msi.wxs

```diff
@@ -5,6 +5,17 @@
         <Media Id="1" Cabinet="product.cab" EmbedCab="yes"/>
 
         <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="CommonAppDataFolder">
+                <Directory Id="ROOTDIR" Name="ContrailVrouterAgent">
```

Comment:
This directory path is based on: https://docs.google.com/presentation/d/1s0S0yQloM7aFx4TxE2nidTz8gdQ5LWaD9GZXYMJM_pY/edit#slide=id.g25279196e7_1_145

Reply:
Now that I think about it, this dir layout it kinda doesn't make sense.
/etc, /var... are subdirs of ContrailVrouterAgent, but potentially other Contrail services (like NodeManager, DockerDriver) could also use /etc and /var. Should we have additional /etc, /var... under NodeManager/ and DockerDriver? I don't think so...

I think we should have something like Program Data/Contrail/{var},{etc}

What do you think?

Reply:
I think it is a good idea. I'll change those paths.